include/caffe/parallel.hpp

```diff
@@ -94,6 +94,9 @@ class P2PSync : public GPUParams<Dtype>, public Solver<Dtype>::Callback,
   }
 
   void run(const vector<int>& gpus);
+  void prepare(const vector<int>& gpus,
+               vector<shared_ptr<P2PSync<Dtype> > >* syncs);
+  inline const int GetInitIter() const { return initial_iter_; }
```

Comment:
And this should be `initial_iter`, or rename `initial_iter_` to `init_iter_` and call it `init_iter`. See other getters throughout Caffe; they are `underscore_case` and do not start with `Get` and match the member name.